app/assets/locales/en.json

```diff
@@ -110,7 +110,7 @@
     "delete_room": "Delete Room",
     "create_new_room": "Create New Room",
     "enter_room_name": "Enter the room name",
-    "shared_by": "shared by ",
+    "shared_by": "shared by",
```

Comment:
Good Point! (Although I would be somewhat surprised if there was a language with different ordering in this case, but I would be similarly surprised in the Last Session example ;-) )

Reply:
@farhatahmad Also it seems like with that solution it would not be possible to emphasize the owner name with <strong>, which I think looks pretty nice.
BTW, I'll go ahead and push an update with whitespace removed from non-english locales which had a whitespace in it. But I'm not sure how you handle the sync with transifex in such cases.